### DISCUSSION TITLE
Validate Documents jdtls popup

### DISCUSSION THREAD

**Discussion by debsishu** [LazyVim/LazyVim#1439] [Q&A] [answered]
Hi, I'm new to neovim. I've setup java using `{ import = "lazyvim.plugins.extras.lang.java" },`
Everything seems to work fine
But whenever I type anything the jdtls shows "validate documents" 
It's not a big deal. But kind of annoying. Can anyone tell me how to disable it just for java?
![Screenshot from 2023-09-11 22-49-34](https://github.com/LazyVim/LazyVim/assets/61660793/ed5846cf-04b3-4929-9553-fc40cb1aa835)

**Comment by dpetka2001** [comment]
I believe you could use `noice` to hide these messages. See this [recipe](https://github.com/folke/noice.nvim/wiki/Configuration-Recipes#hide-written-messages-1) to get started and change it to your needs.

**Reply by debsishu** [reply]
I've added this as you suggested.
```lua
return {
  {
    "folke/noice.nvim",
    require("noice").setup({
      routes = {
        {
          filter = {
            event = "msg_show",
            kind = "",
            find = "written",
          },
          opts = { skip = true },
        },
      },
    }),
  },
}
```

But it did not work. Do I have to change something?

**Reply by dpetka2001** [reply]
In the `find` field you have to declare what the message you want to find is. In your case it might be `"jdtls"`? I haven't done this, so I can't say for sure, but that's what it seems to me.

**Reply by debsishu** [reply]
adding `"jdtls"` did not work also.

**Reply by dpetka2001** [reply, marked as answer]
Try
```lua
routes = {
        {
          filter = {
            event = "lsp",
            kind = "progress",
            find = "jdtls",
          },
          opts = { skip = true },
        },
      },
```

**Reply by debsishu** [reply]
It also did not work😅.

**Reply by dpetka2001** [reply]
Did you put `kind = "progress"` because I edited my post? If that doesn't work try with `kind = "message"`

**Reply by debsishu** [reply]
It worked. Thanks.
